src/main/java/com/dislinkt/post/mapper/PostMapper.java

```diff
@@ -0,0 +1,40 @@
+package com.dislinkt.post.mapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.dislinkt.post.dto.PostDTO;
+import com.dislinkt.post.model.Post;
+
+public class PostMapper implements MapperInterface<Post, PostDTO> {
+
+    @Override
+    public Post toEntity(PostDTO dto) {
+        return new Post(
+            dto.getId(), 
+            dto.getText(), 
+            dto.getImageUrl(), 
+            dto.getLinks());
+    }
+
+    @Override
+    public List<Post> toEntityList(List<PostDTO> dtoList) {
+        return dtoList.stream().map(this::toEntity).collect(Collectors.toList());
+    }
+
+    @Override
+    public PostDTO toDto(Post entity) {
+        return new PostDTO(
+            entity.getId(), 
+            entity.getText(), 
+            entity.getImageUrl(), 
+            entity.getLinks(), 
+            entity.getPerson().getId());
```

Comment:
Think about this from an FE perspective, do you need a Person's Id or name? Or maybe both of them?

Reply:
I don't think a person's name is needed, because the person can be found through id alone.